texts/formatted_medtner.txt

```diff
@@ -0,0 +1,1665 @@
+"<gives>"
+	"give" vblex pres p3 sg
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<date of birth>"
+	"date# of birth" n sg
+"<as>"
+;	"as" adv
+;	"as" preadv
+;	"as" cnjsub
+;	"as" cnjadv
+	"as" pr
+"<5>"
+	"5" num
+"<January>"
+	"January" n sg
+"<1880>"
+	"1880" num
+"<..>"
+	".." sent
+
+
+"<Medtner>"
+	"*Medtner"
+"<first>"
+	"first" adv
+;	"first" adj ord
+"<took>"
+	"take" vblex past
+"<piano>"
+;	"piano" adv
+	"piano" n sg
+"<lessons>"
+	"lesson" n pl
+"<from>"
+	"from" pr
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<mother>"
+	"mother" n sg
+;	"mother" vblex inf
+;	"mother" vblex pres
+;	"mother" vblex imp
+"<until>"
+	"until" cnjsub
+;	"until" cnjadv
+;	"until" pr
+"<the>"
+	"the" det def sp
+"<age>"
+	"age" n sg
+;	"age" vblex inf
+;	"age" vblex pres
+;	"age" vblex imp
+"<of>"
+	"of" pr
+"<ten>"
+	"ten" num pl
+;	"ten" n sg
+"<.>"
+	"." sent
+"<He>"
+;	"He" np cog sg
+	"prpers" prn subj p3 m sg
+"<also>"
+	"also" adv
+"<had>"
+;	"have" vbhaver past
+	"have" vblex past
+"<lessons>"
+	"lesson" n pl
+"<from>"
+	"from" pr
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<mother>"
+	"mother" n sg
+;	"mother" vblex inf
+;	"mother" vblex pres
+;	"mother" vblex imp
+"<'s>"
+	"'s" gen
+;	"be" vbser pres p3 sg
+;	"have" vbhaver pres p3 sg
+"<brother>"
+	"brother" n sg
+"<Fyodor>"
+	"*Fyodor"
+"<Goedicke>"
+	"*Goedicke"
+"<(>"
+	"(" lpar
+"<the>"
+	"the" det def sp
+"<father>"
+	"father" n sg
+;	"father" vblex inf
+;	"father" vblex pres
+;	"father" vblex imp
+"<of>"
+	"of" pr
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<more>"
+;	"more" adv
+	"more" preadv
+;	"more" n sg
+;	"more" det qnt sp
+"<famous>"
+	"famous" adj
+"<cousin>"
+	"cousin" n sg
+"<Alexander>"
+;	"Alexander" np cog sg
+	"Alexander" np ant m sg
+"<Goedicke>"
+	"*Goedicke"
+"<)>"
+	")" rpar
+"<.>"
+	"." sent
+"<Then>"
+;	"then" adv
+	"then" cnjadv
+"<he>"
+	"prpers" prn subj p3 m sg
+"<entered>"
+;	"enter" vblex pp
+	"enter" vblex past
+"<the>"
+	"the" det def sp
+"<Moscow>"
+	"Moscow" np top sg
+"<Conservatory>"
+	"conservatory" n sg
+"<.>"
+	"." sent
+"<He>"
+;	"He" np cog sg
+	"prpers" prn subj p3 m sg
+"<graduated>"
+	"graduate" vblex past
+;	"graduate" vblex pp
+"<in>"
+;	"in" adv
+	"in" pr
+"<1900>"
+	"1900" num
+"<at>"
+	"at" pr
+"<the>"
+	"the" det def sp
+"<age>"
+	"age" n sg
+;	"age" vblex inf
+;	"age" vblex pres
+;	"age" vblex imp
+"<of>"
+	"of" pr
+"<20>"
+	"20" num
+"<,>"
+	"," cm
+"<taking>"
+;	"take" vblex pprs
+	"take" vblex ger
+;	"take" vblex subs
+;	"taking" n sg
+"<the>"
+	"the" det def sp
+"<Anton>"
+	"Anton" np ant m sg
+"<Rubinstein>"
+	"*Rubinstein"
+"<prize>"
+	"prize" n sg
+;	"prize" vblex inf
+;	"prize" vblex pres
+;	"prize" vblex imp
+"<,>"
+	"," cm
+"<having>"
+;	"have" vbhaver pprs
+	"have" vbhaver ger
```

Comment:
Is this right?

Reply:
Yes. It is the continuous tense, and there is another verb directly afterwards